lib/layout.js

```diff
@@ -1,11 +1,11 @@
 import {
-    HttpIncoming,
+    HttpIncoming as HttpIncomingImpl,
     template,
     isFunction,
     pathnameBuilder,
     uriIsRelative,
-    AssetCss,
-    AssetJs,
+    AssetCss as AssetCssImpl,
+    AssetJs as AssetJsImpl,
```

Comment:
Oh this is all kinds of yuck.

Reply:
Yeah I don't love it, but the `@typedef` and the `import` collide :(